erasure-readfile.go

```diff
@@ -262,14 +262,17 @@ func erasureReadFile(writer io.Writer, disks []StorageAPI, volume string, path s
 		}
 
 		var outSize, outOffset int64
+
+		// Total data to be read.
+		outSize = blockSize
```

Comment:
> We need to think about moving this logic into a separate function or something. That way we can test thoroughly this skipping logic.

true, will check on this. Also there are so many variables in erasureReadFile() it's difficult to read, will see how it can be improved.